tools/alerts/create_alerts.py

```diff
@@ -1,16 +1,19 @@
 #!/usr/bin/env python3
 
+from __future__ import annotations
```

Comment:
Do you mind filing an issue to get rid of those `from __future__ import annotations` as we are about to deprecated Python-3.8 soon and 3.9 already supports those